### ISSUE TITLE
Build Error: SyntaxError: Unexpected token import - `import axios from 'axios';`

### ISSUE THREAD

**Issue by dwilt** [nozzle/react-static#511]
When I initially create a new app (`react-static create`) and then use the [redux](https://github.com/nozzle/react-static/tree/master/examples/redux) template, everything works fine. When I try to install [Butter CMS](https://buttercms.com/docs/api-client/react) and `import` it into `static.config.js`, I get an ES6/Babel Syntax Error:

![image](https://user-images.githubusercontent.com/1065539/37774826-9330f958-2de1-11e8-8ef5-3b38ff6a7cb3.png)

What's crazy is that if I back out all the code I just did relating to Butter CMS, I still get the error. I have to create a brand new project to get back to being able to work again. I created a video showing the problem:

[![react-static-import-bug](https://img.youtube.com/vi/Iy6W-7XdGPU/0.jpg)](https://www.youtube.com/watch?v=Iy6W-7XdGPU)

**Comment by dwilt** [comment]
I was able to prevent having to create a new project it by uninstalling Butter (`npm uninstall buttercms --save`). However, it just being in the `package.json` shouldn't affect the build, right?

**Comment by tannerlinsley** [comment]
Could there be some clash with peer dependencies?

**Comment by dwilt** [comment]
How would I check that?

**Comment by JustFly1984** [comment]
When you do `npm i`, it usually logs out all peer dependency collisions.

2018-03-24 1:22 GMT+08:00 Dan Wilt <notifications@github.com>:

**Comment by tannerlinsley** [comment]
I would try removing the node modules directory and killing any lock files.
Then reinstalling. This might solve your problem.

**Comment by tannerlinsley** [comment]
Closing for triage, reopen if problem persists

**Comment by abroccoli** [comment]
Hello, I am encountering this error while trying to use foundation-sites with react-static. I used npm to install `foundation-sites@6.4.4-rc1` and I am running `react-static@5.7.1`. I haven't made any changes to the default webpack configuration in `static.config.js` and I am importing foundation-sites at the top of the component that requires it `import 'foundation-sites';`. 

Everything works fine running a local node server using `npm start`, however when I try to build with `npm run build` I get the error:

```
Error: /Users/abroccoli/Documents/apm.net/apm-react-static/node_modules/foundation-sites/dist/js/npm.js:1
  (function (exports, require, module, __filename, __dirname) { import $ from 'jquery';
                                                                ^^^^^^
  SyntaxError: Unexpected token import
```
I've tried to find peer dependency collisions with `npm i` and none were logged, I've also tried removing the node_modules directory and the package-lock.json file, reinstalling all node modules, and still the error persists.

Am I missing a step somewhere?
